src/ParserParameterProcessor.php

```diff
@@ -154,6 +152,18 @@ public function addParameter( $key, $value ) {
 	}
 
 	/**
+	 * @since 2.3
+	 *
+	 * @param string $key
+	 * @param array $values
+	 */
+	public function setParameter( $key, array $values ) {
```

Comment:
At first I thought this name was wrong. But it's the existing `addParameter` that is wrong and should be something like `addParameterValue` or `addToParameter`